### DISCUSSION TITLE
(Poll) can we please update the layout animation docs?

### DISCUSSION THREAD

**Discussion by iway1-hstk** [software-mansion/react-native-gesture-handler#2135] [Polls]
Current docs for layout animations: https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/layout_animations

The tutorial here says to use the `<AnimatedLayout/>` component, which no longer exists. I'm not saying we need a new video, but please at least take this one down?

Also, anyone know how layout animations actually work now? Can't find anything in the docs. Tried running `enableLayoutAnimations(true)` and using the layout prop without `<AnimatedLayout/>`, doesn't work though

**Comment by j-piasecki** [comment]
Hi! I can pass it along to the Reanimated team, but I think you should move the discussion to [their repository](https://github.com/software-mansion/react-native-reanimated) anyways 🙂.
